gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
 #Mon Mar 28 16:42:31 CEST 2022
 distributionBase=GRADLE_USER_HOME
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.2-bin.zip
```

Comment:
Is not a good idea to update gradle like this. Need a separate PR, approve PR, merge in dev. And only when all the devs had the latest release with updated plugins continue normal development, to avoid misalignment and problems during the development process.